reviews/utils/review_actions.py

```diff
@@ -210,3 +212,67 @@ def action_url(self):
     def description(self):
 
         return _('Verander aangestelde commissieleden')
+    
+class SendConfirmation(ReviewAction):
+
+    def is_available(self):
+
+        user = self.user
+        review = self.review
+
+        '''Only the secretary is able to send the confirmation letter and/or change the date.
+        The review needs to be closed and have an approved status.'''
+
+        user_groups = user.groups.values_list("name", flat=True)
+        if not settings.GROUP_SECRETARY in user_groups:
+            return False
+        
+        if review.stage < review.CLOSED:
+            return False
+        
+        if not review.continuation in [review.GO, review.GO_POST_HOC]:
+            return False
+        
+        return True
+    
+    def action_url(self):
+
+        return reverse('proposals:confirmation', args=(self.review.pk,))
+    
+    def description(self):
+        send_letter = _('Bevestigingsbrief versturen')
+        change_date = _('Datum van bevestigingsbrief aanpassen')
+        return send_letter if self.review.proposal.date_confirmed is None else change_date
+    
+'''
+This class should lead to the archive_hide url, but the hide functionality does currently not work properly/ is not in use.
+Therefore it is commented out in the ReviewActions class.
+'''
```

Comment:
This should be moved to right after `class HideReview(ReviewAction):`, as it's a docstring

Reply:
Alternatively, as the contents are not really docstring like, it should be changed to a normal comment (using #)

In general, using `'''` strings are only used for docstrings or multiline string values